python/python/bystro/search/index/bystro_file.pyx

```diff
@@ -95,12 +95,12 @@ cdef class ReadAnnotationTarball:
 
                             overlap_values.append(overlap_value)
 
-                        values.append(overlap_values)
+                        if len(overlap_values) == 1:
```

Comment:
honest question: I'm not sure I have enough context here but I think in general I'd opt for collapsing this ambiguity in favor of keeping the data type consistent whether the list has one or many elements, so that values will always be a list of lists of overlap_values (some of which may be singletons).  Otherwise the consuming code will have to make an isinstance check and branch appropriately.

(But if we've considered that alternative and it has other drawbacks that make this the best option, it's a non-blocking concern)

Reply:
I don't feel strongly either way; let's go that route.